product-service/serverless.ts

```diff
@@ -0,0 +1,94 @@
+import type { AWS } from '@serverless/typescript';
+
+import getProductsList from '@functions/getProductsList';
+import getProductsById from '@functions/getProductsById';
+import createProduct from '@functions/createProduct';
+
+const serverlessConfiguration: AWS = {
+  service: 'product-service-natallia-adziyanava',
+  frameworkVersion: '3',
+  plugins: ['serverless-auto-swagger','serverless-esbuild', 'serverless-offline'],
+  provider: {
+    name: 'aws',
+    runtime: 'nodejs14.x',
+    region: 'eu-central-1',
+    stage: 'dev',
+    apiGateway: {
+      minimumCompressionSize: 1024,
+      shouldStartNameWithService: true,
+    },
+    iam: {
+      role: {
+        permissionsBoundary: 'arn:aws:iam::${aws:accountId}:policy/eo_role_boundary',
+        statements: [{
+          Effect: 'Allow',
+          Action: [
+            's3:*',
+            'dynamodb:Query',
```

Comment:
It looks like this file is completely new, but it should not be new. When each next task is done, you can merge this branch to master and then create a branch for next task from master